Resources/Prototypes/Entities/Mobs/NPCs/human.yml

```diff
@@ -39,3 +39,14 @@
       types:
         Blunt: 200
 
+- type: entity
+  parent: MobHuman
+  id: MobCluwne
+  name: person
+  noSpawn: true
+  description: A polymorphed unfortunate.
+  suffix: do not spawn.
```

Comment:
same thing i said about noSpawn, but this time dont put a suffix

Reply:
This should be fixed now, removed suffix since it has noSpawn